### ISSUE TITLE
How you generate Events

### ISSUE THREAD

**Issue by milutinke-crowdbotics** [atinm/fpdb#2]
Hey, This is more a question than the issue, because I could not find any email of any contributors.

My question is: Are you caclculating here just Statistics for example: VPIP in %, PFR in %, or you have game actions data (each game action that happened) and you first figure out if user `volontery put money in pot` or `is user pre flop raiser` and etc. and than you do statistics.

Why i ask this: We are developing some poker game, and we have `game actions` for example: `player_join`, `game_started`, `raise`, `call` etc, and I need to find based on this actions that are coming `one by one` is user is VPIP or PFR at that hand?

So to sum up: Are you also figuring out this VPIP, PFR... events based on game history (or live data), or this supported applications already calculate that and you just do analysis.

If you are figuring out which events happened, can you point out to me to the part of code that do that please? 
Thanks
